openedx/core/djangoapps/credentials/settings/common.py

```diff
@@ -0,0 +1,6 @@
+"""Common settings for Credentials."""
+
+
+# def plugin_settings(settings):
```

Comment:
Same question: are there settings for this?  If so we should remove them from the giant common.py and add them here.